### ISSUE TITLE
CD8 x-axis refinement, focus on obvious double populations

### ISSUE THREAD

**Issue by jlanej** [PankratzLab/auto-fcs#69]
#63 did not perform that well, will try with a similar trick used in #67

**Comment by jlanej** [comment]
oddball 
![2016-11-29_panel_1_hb_group_one_f1652719_016 fcs cytotoxic_tcells-cd8](https://user-images.githubusercontent.com/6052453/33144620-47eee08e-cf83-11e7-87bb-9c3d8da7d216.png)

**Comment by jlanej** [comment]
<img width="886" alt="screen shot 2017-11-22 at 12 50 12 pm" src="https://user-images.githubusercontent.com/6052453/33144771-bb76ce4a-cf83-11e7-9bf1-97f7e7da4463.png">

**Comment by jlanej** [comment]
This had an old-gate annotation, but not sure if it would be worth adjusting for
![2017-01-10_panel_1_hb_group_one_f1636024_006 fcs cytotoxic_tcells-cd8](https://user-images.githubusercontent.com/6052453/33147991-a25cd3ee-cf8f-11e7-8770-36521526e92e.png)

**Comment by jlanej** [comment]
Also old-gate annotated, and worth adjusting
<img width="886" alt="screen shot 2017-11-22 at 2 18 54 pm" src="https://user-images.githubusercontent.com/6052453/33148110-21ba6840-cf90-11e7-9289-271f196257ed.png">

**Comment by jlanej** [comment]
Testing a better default range for mindensity
<img width="886" alt="screen shot 2017-11-22 at 2 23 05 pm" src="https://user-images.githubusercontent.com/6052453/33148344-23b1543c-cf91-11e7-892a-9483921a4865.png">

**Comment by jlanej** [comment]
Current version uses mindensity with an updated range that will default to the max of that range when two peaks are detected. Also, increased the smoothing parameter to avoid situations like the above. As the above was favored to the "old-gate", the current method also does not trim the 2 examples below since there are not two obvious populations.

<img width="886" alt="screen shot 2017-11-22 at 3 00 40 pm" src="https://user-images.githubusercontent.com/6052453/33149603-f06bcbac-cf95-11e7-90ca-a2f54139ea7f.png">

<img width="886" alt="screen shot 2017-11-22 at 2 54 40 pm" src="https://user-images.githubusercontent.com/6052453/33149366-1a87d62a-cf95-11e7-91b5-5019431ffb6b.png">

But, situations like below will (hopefully) be trimmed. 
<img width="886" alt="screen shot 2017-11-22 at 3 04 24 pm" src="https://user-images.githubusercontent.com/6052453/33149770-887086ea-cf96-11e7-841d-c5125c82b799.png">

<img width="886" alt="screen shot 2017-11-22 at 2 56 47 pm" src="https://user-images.githubusercontent.com/6052453/33149778-913fe496-cf96-11e7-8119-d36120f07bca.png">

We will also incorporate the boolean gate, so that the below will be reverted to the "old-gate" if the new method extends too far, such as the case here

<img width="886" alt="screen shot 2017-11-22 at 3 13 06 pm" src="https://user-images.githubusercontent.com/6052453/33150089-b3391422-cf97-11e7-8d26-0a75cb63f3c0.png">

and here

<img width="886" alt="screen shot 2017-11-22 at 3 30 32 pm" src="https://user-images.githubusercontent.com/6052453/33150672-2a76a44e-cf9a-11e7-9b7d-de6083edaa72.png">

TL;DR 
Trying to focus on trimming only when the extra cluster is obvious, otherwise default to the original gate (or close to it)
